artifactory/services/utils/deleteutils.go

```diff
@@ -47,7 +47,7 @@ func WriteCandidateDirsToBeDeleted(candidateDirsReaders []*content.ContentReader
 	}
```

Comment:
This function is already unit-tested, but I suggest we enhance its unit test, so that it also covers the cases which this PR fixes.

Reply:
Done.